app/src/main/res/layout/fragment_record.xml

```diff
@@ -8,18 +8,41 @@
     android:paddingBottom="?attr/actionBarSize"
     tools:context=".fragments.RecordFragment">
 
-    <com.google.android.material.floatingactionbutton.FloatingActionButton
-        android:id="@+id/btnRecord"
+
+    <LinearLayout
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_gravity="bottom|center_horizontal"
-        android:layout_marginBottom="16dp"
         android:layout_alignParentBottom="true"
         android:layout_centerHorizontal="true"
-        app:backgroundTint="@color/primary"
-        app:elevation="6dp"
-        app:fabSize="normal"
-        app:srcCompat="@drawable/ic_mic_white_36dp" />
+        android:paddingBottom="16dp"
+        android:paddingHorizontal="5dp"
+        android:clipToPadding="false"
+        android:layout_gravity="bottom|center_horizontal"
+        android:orientation="horizontal">
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/btnPause"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:layout_marginHorizontal="16dp"
+            android:fontFamily="sans-serif-condensed"
+            app:icon="@drawable/ic_media_pause"
+            android:text="@string/pause_recording_button"
+            android:textAllCaps="true"
+            app:elevation="2dp" />
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/btnRecord"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:layout_marginHorizontal="16dp"
+            app:backgroundTint="@color/primary"
+            app:elevation="6dp"
```

Comment:
for some reason, `app:elevation` has no effect after this change.
to simulate a small screen, I'm using a split screen for testing. please take a look at the screenshots below.

BEFORE your changes (`master` branch)

1
![image](https://user-images.githubusercontent.com/3858548/197399785-7c9e1243-3bed-4c7c-a812-5d6ccd2357dc.png)

2
![image](https://user-images.githubusercontent.com/3858548/197399802-74a11397-7240-44fe-b2d3-ac851d3b383f.png)

3
![image](https://user-images.githubusercontent.com/3858548/197399826-e3134500-ea24-4569-ad2b-a2f0753dd668.png)

AFTER your changes

1
![image](https://user-images.githubusercontent.com/3858548/197399857-bfc5d55f-3cd1-470d-b874-eb9f04c59ace.png)

2
![image](https://user-images.githubusercontent.com/3858548/197399874-d68b75da-722f-4e77-99db-a97378430891.png)

3. no photo

as you can see, there's a visual regression. RECORD/STOP and PAUSE/RESUME buttons are now overlapping with the circular progress bar. But buttons should be on top.